tools/proofers/preview.js

```diff
@@ -26,16 +26,26 @@ function processExMath(text, textFunction, allowMath) {
     }
 }
 
-$(function () {
-    function removeComments(textLine) {
-        return textLine.replace(/\[\*\*[^\]]*\]/g, '');
-    }
-
-    // true if txtLine contains anything except only comments or spaces
-    function nonComment(textLine) {
-        return (/\S/.test(removeComments(textLine)));
+// find index of next unmatched ] return 0 if none found
+function findClose(txt, index) {
+    var result;
```

Comment:
Should we use let and const like our other js files?

Reply:
Yes.